wolfCLU/src/tools/wolfsslFuncs.c

```diff
@@ -68,77 +68,83 @@ int     i          =   0;       /* loop variable */
 void wolfsslVerboseHelp()
 {
     printf("\nwolfssl Command Line Utility version %3.1f\n\n", VERSION);
+
     /* hash options */
     const char* algsenc[] = {        /* list of acceptable algorithms */
 #ifndef NO_MD5
-            "md5"
+        "md5"
 #endif
 #ifndef NO_SHA
-                ,"sha"
+        ,"sha"
 #endif
 #ifndef NO_SHA256
-                ,"sha256"
+        ,"sha256"
 #endif
 #ifdef WOLFSSL_SHA384
-                ,"sha384"
+        ,"sha384"
 #endif
 #ifdef WOLFSSL_SHA512
-                ,"sha512"
+        ,"sha512"
 #endif
 #ifdef HAVE_BLAKE2
-                ,"blake2b"
+        ,"blake2b"
 #endif
-        };
+#ifndef NO_CODING
+        ,"base64enc"
```

Comment:
Logic in wolfsslHashSetup was:

```
#ifndef NO_CODING
     #ifdef WOLFSSL_BASE64_ENCODE
         , "base64enc"
     #endif
         , "base64dec"
 #endif
```

Is there a reason we aren't checking for WOLFSSL_BASE64_ENCODE here?